lib/foreman.rb

```diff
@@ -45,4 +45,24 @@ def self.input_types_registry
   def self.settings
     SettingRegistry.instance
   end
+
+  DOWNLOAD_UTILITIES =
```

Comment:
@goarsna, I'm so sorry for suggesting you to make it a constant even though it makes sense... We're going to switch into Zeitwerk, which ideally wants a separate file for each constant, so let's not create a nested constant, which will be used globally within this file. Just to make thing simpler, let's rather convert it into a methods whenever you're going to make another change within the PR.

Reply:
No problem :) But would `lib/foreman.rb` still be the correct place to store the `download_utilities` method?

Reply:
Unless someone else objects, yeap.